crates/turbopack-ecmascript/src/references/mod.rs

```diff
@@ -725,6 +730,52 @@ pub(crate) async fn analyze_ecmascript_module(
                         )
                     }
 
+                    JsValue::WellKnownFunction(WellKnownFunctionKind::RequireContext) => {
+                        let args = linked_args(args).await?;
+                        if (1..=3).contains(&args.len()) {
+                            let options = match parse_require_context(&args) {
+                                Ok(options) => options,
+                                Err(reason) => {
+                                    let (args, hints) = explain_args(&args);
+                                    handler.span_err_with_code(
+                                        span,
+                                        &format!(
+                                            "require.context({args}) is not statically \
+                                             analyse-able: {reason}{hints}",
```

Comment:
Unrelated to this PR: we should switch to american english: https://linear.app/vercel/issue/WEB-890/convert-turbopack-errors-and-messaging-to-american-english